src/components/Main.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+    <div class="container">
+        <div class="section">
+            <div class="columns">
+                <div class="column is-8 is-offset-2">
+                    <span class = "wrapper">
+                        <div class = "readoutContainer" v-bind:style ="{float: floatVal, width: widthVal}">
+                            <div class="readout">
+                                <div v-if="!rendered">
+                                    <textarea
+                                        v-model="inputText"
+                                        class="textarea text-input"
+                                        placeholder="Paste your letter of recommendation here."
+                                    ></textarea>
+                                    <div class="btn-container">
+                                        <button
+                                            class="button is-info is-fullwidth submit-button"
+                                            v-on:click="renderIssues" 
+                                            @click = "showSidebar(); hideHeader(); changeWidth()"
+                                        >
+                                            Submit
+                                        </button>
+                                    </div>
+                                </div>
+                                <div v-if="rendered">
+                                    <button
+                                        class="button is-primary is-fullwidth back-button"
+                                        v-on:click="
+                                            () => {
+                                            rendered = false;
+                                            }
+                                        "
```

Comment:
Can you write a named function with this one line of code so we don't get this ugly formatting?